docs/datagrid/datagrid-v2-migration.md

```diff
@@ -21,13 +21,126 @@ const grid = DataGrid.dataGrid('container', {
 });
 ```
 
+or
+
+```js
+const grid = DataGrid.dataGrid('container', {
+    table: {
+        columns: { ... }
+    }
+});
+```
+
 ### Columns
-The `useHTML` parameter has been moved to the `columns` object.
+It is worth to mantion that the `useHTML` parameter has been moved to the `columns` object.
 
 You can find more information in our [API Reference](https://api.highcharts.com/dashboards/typedoc/interfaces/DataGrid_DataGridOptions.IndividualColumnOptions.html#useHTML).
 
 ### Styling
-The name of CSS classes have been also changed. We recommend to read our [article about styling](https://www.highcharts.com/docs/datagrid/style-by-css) the DataGrid 2.0.
+The main change is CSS styling instead of the build-in API options.
```

Comment:
I would also mention that styles are even more important now because they replace the old JS options and should be treated equally as important as the API.